docs/troubleshooting/installation.md

```diff
@@ -106,6 +106,46 @@ $ sudo yq eval .token /etc/rancher/rancherd/config.yaml
 
 :::
 
+## Check the status of Harvester components
+
+Before checking the status of Harvester components, obtain a copy of the Harvester cluster's kubeconfig file using either of the following methods:
+
+- On the Harvester UI, go to the **Harvester Support** screen and then click **Download KubeConfig**.
+
+- Run the following commands on any of the management nodes:
+  ```shell
```

Comment:
`sudo -i cat /etc/rancher/rke2/rke2.yaml`

gets the result and still on non-root mode